pkg/kv/cosmosdb/store.go

```diff
@@ -441,7 +448,7 @@ func (e *EntriesIterator) Entry() *kv.Entry {
 }
 
 func (e *EntriesIterator) Err() error {
-	return e.err
+	return convertError(e.err)
```

Comment:
think we should convert when we get back from API calls - but when we keep the converted error.